drivers/ovsdbDriver.go

```diff
@@ -217,7 +217,7 @@ func (d *OvsdbDriver) createDeleteBridge(bridgeName, failMode string, op oper) e
 	}
 
 	// simple insert/delete operation
-	brOp := libovsdb.Operation{}
+	var brOp libovsdb.Operation
```

Comment:
If I understood the golang right, the above two statements have the same effect. Was this flagged by the tool? Just curious.